citation/management/commands/flagged_faulty_data_publication.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from citation import models
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Flagging publication that doesnt have either of ODD or Other Narrative selected in model documentation"
+
+    def handle(self, *args, **options):
+        narrative_list = list()
+        for category in models.ModelDocumentation.CATEGORIES:
+            if category['category'] == 'Narrative':
+                for name in category['modelDocumentationList']:
+                    narrative_list.append(name['name'])
+
+        total = models.Publication.api.primary(status='REVIEWED')
+        difference = set(total).difference(
```

Comment:
why not turn this into a single query:

`Publication.api.primary(status='REVIEWED').exclude(model_documentation__name__in=narrative_list)`